### ISSUE TITLE
do you have any example training data set ??

### ISSUE THREAD

**Issue by Junhyuk** [BartyzalRadek/Multi-label-Inception-net#16]
Please comment if you have them.

**Comment by tianyuecao** [comment]
I think you can try some open database, such as MS COCO or VOC.

**Comment by Easyfeng222** [comment]
@tianyuecao Is it necessary to modify the code with this, the ＭＳ　ＣＯＣＯ　format and the data set of the project are not the same?I look forward to hearing from you, thank you in advance.

**Comment by tianyuecao** [comment]
> @tianyuecao Is it necessary to modify the code with this, the ＭＳ　ＣＯＣＯ　format and the data set of the project are not the same?I look forward to hearing from you, thank you in advance.

Not exactly. I'm now using the objects in each image as the labels, thus I have to extract it from the annotation file. But i'm now using torch to implement it, maybe i'll write a dataloader for MS COCO some time :)

**Comment by Easyfeng222** [comment]
@tianyuecao What is the accuracy of the MS COCO dataset you tested? ? This project is a txt for each picture, then does it have a txt for the MS COCO data set?

**Comment by Easyfeng222** [comment]
@tianyuecao I am researching multi-label image classification, if you have the chance, we can add a WeChat exchange.　ｕpFF222

**Comment by tianyuecao** [comment]
> @tianyuecao What is the accuracy of the MS COCO dataset you tested? ? This project is a txt for each picture, then does it have a txt for the MS COCO data set?

Actually I didn't apply it on MS COCO. I've tested it on a dataset from my lab. You can extract txt files for MS COCO from its annotation files if you'd like to.

**Comment by tianyuecao** [comment]
> @tianyuecao I am researching multi-label image classification, if you have the chance, we can add a WeChat exchange.　ｕpFF222

I'd like to, maybe leave me an email?

**Comment by Easyfeng222** [comment]
@tianyuecao １７３８８３４３６＠qq.com
